### PR TITLE
add entry for omero-web release

### PR DESCRIPTION
**Pull request by jburel**
Add entry for release instructions of OMERO.web

### CODE REVIEW

File: docs/omero-web-release-process.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Release process
+^^^^^^^^^^^^^^^
+
+Source code release
+-------------------
+
+.. seealso::
+
+    :doc:`python-development`
+       Release process of Python components
```
Comment (sbesson) [inline docs/omero-web-release-process.rst:20]: Alternatively this could (also ?)link to https://github.com/ome/omero-web#release-process.

Comment (jburel) [reply docs/omero-web-release-process.rst]: done

**Review by sbesson — APPROVED** [review]
The page makes sense to me and is consistent with the release process page introduced for OMERO.server. 

A few thoughts which should not hold the merging of this PR

- the Docker section could likely include a paragraph as in the OMERO.server page indicating which repository should be tagged (I assume there is no version to maintain for this image)
- is there any form of documentation update for an OMERO.web release?
- the format of the announcement post lacks consistency with the one in OMERO.server
- as more release process pages are migrated to the contributing documentations and entries are added to the left hand panel, we will reach the point where organising these pages with nested layout might increase the readability

**Comment by jburel** [comment]
I follow the format used by @knabar i.e. usage of dots. 
We can modify that so all the announcements follow the same pattern

**Comment by jburel** [comment]
> The page makes sense to me and is consistent with the release process page introduced for OMERO.server.
> 
> A few thoughts which should not hold the merging of this PR
> 
> * the Docker section could likely include a paragraph as in the OMERO.server page indicating which repository should be tagged (I assume there is no version to maintain for this image)
I have added a link to the repo, no version to update. This might change if https://github.com/ome/omero-web-docker/pull/78 is merged
> * is there any form of documentation update for an OMERO.web release?
Not really
> * the format of the announcement post lacks consistency with the one in OMERO.server
Comment above
> * as more release process pages are migrated to the contributing documentations and entries are added to the left hand panel, we will reach the point where organising these pages with nested layout might increase the readability

I will look into that in a follow-up PR

**Pull request merged**